### PR TITLE
Attempt to fix a crash when tracking scrollview

### PR DESCRIPTION
**Pull request by muukii**

### CODE REVIEW

File: Rideau/RideauInternalView.swift [hunk 1]
```diff
@@ -565,8 +568,10 @@ final class RideauInternalView : RideauTouchThroughView {
           heightConstraint.constant -= offset
         }
       }
-      
-      lastOffset = targetScrollView?.contentOffset
+
+      if let scrollView = targetScrollView {
+        lastOffset = scrollView.contentOffset
+      }
```
Comment (muukii) [inline Rideau/RideauInternalView.swift:574]: I thought it inserts nil with targetScrollView is nil.

**Comment by muukii** [comment]
To fix this issue with another approach : https://github.com/muukii/Rideau/pull/32

**Comment by ntnmrndn** [comment]
Just got the crash again :D Can we merge it ?

**Comment by muukii** [comment]
OMG

ok I'll merge it and then investigate it later.

**Pull request merged**